tools/font-edit/README.md

```diff
@@ -3,15 +3,50 @@
 which are expected to fit the requirements of embedded systems with larger
 screens.
 
+<p align="center">
+  <img src="./figs/demo.gif" />
+</p>
+
 ## Build Dependency
 ```shell
-sudo apt-get install libx11-dev libcairo2-dev
+sudo apt-get install libsdl2-dev libcairo2-dev
 ```
 
 ## Usage
 ```shell
 make
 ./twin-fedit < nchars
 ```
+### Key bindings
+
+
+| Key binding | Function |
```

Comment:
Replace "Key binding" with "Key". The so-called key-binding means the mapping between keys and their functionalities.